pages/common/pip-install.md

```diff
@@ -15,6 +15,10 @@
 
 `pip install -r {{requirements.txt}}`
 
+- Install packages from an URL or local file archive (.tar.gz | .whl):
```

Comment:
```suggestion
- Install packages from an URL or local file package:
```

Reply:
In my opinion archive is more precise than package here.